content/push/configure/device.textile

```diff
@@ -180,41 +162,37 @@ The following diagram demonstrates the direct activation process:
 
 The following example initializes an instance of the Ably Realtime service and then activates the push notifications feature _for that instance_:
 
-```[kotlin]
+```[realtime_kotlin]
 val ably = getAblyRealtime()
 ably.setAndroidContext(context)
 ably.push.activate()
 ```
 
-```[java]
+```[realtime_java]
 AblyRealtime ably = getAblyRealtime();
 ably.setAndroidContext(context);
 ably.push.activate();
 ```
 
-```[swift]
+```[realtime_swift]
 let ably = self.getAblyRealtime()
 ably.push.activate()
 ```
 
-```[objc]
+```[realtime_objc]
 ARTRealtime *ably = [self getAblyRealtime];
 [ably.push activate];
 ```
 
-```[csharp]
+```[realtime_csharp]
 ActivatePush = new Command(() => Ably.Push.Activate());
 ```
 
-```[flutter]
+```[realtime_flutter]
 AblyRealtime ably = getAblyRealtime();
 ably.push.activate();
 ```
 
-```[javascript]
```

Comment:
if we're removing javascript examples from this page entirely then we should also remove the javascript row in the table under "Device activation lifecycle"